### ISSUE TITLE
multicomp.py: all tests with alpha != 0.05 that use allpairtests are wrong, as multipletests in allpairtests is called with wrong argument

### ISSUE THREAD

**Issue by diego-mazon** [statsmodels/statsmodels#5974]
I think that reject, pvals_corrected, alphacSidak, alphacBonf returned by allpairtests are using alpha=0.05 regardless of the passed value to allpairtests, as multipletests in line 912 of multicomp.py is being called with alpha=0.05 rather than alpha=alpha.

Current:
 ``` python
def allpairtest(self, testfunc, alpha=0.05, method='bonf', pvalidx=1):
        res = []
        for i,j in zip(*self.pairindices):
            res.append(testfunc(self.datali[i], self.datali[j]))
        res = np.array(res)
        reject, pvals_corrected, alphacSidak, alphacBonf = \
                multipletests(res[:, pvalidx], alpha=0.05, method=method)
```

Tentative fix:

 ``` python
def allpairtest(self, testfunc, alpha=0.05, method='bonf', pvalidx=1):
        res = []
        for i,j in zip(*self.pairindices):
            res.append(testfunc(self.datali[i], self.datali[j]))
        res = np.array(res)
        reject, pvals_corrected, alphacSidak, alphacBonf = \
                multipletests(res[:, pvalidx], alpha=alpha, method=method)
```

**Comment by bashtage** [comment]
Yes, looks wrong. Can you submit a PR to fix?

**Comment by diego-mazon** [comment]
I think I just did. Hope I did it right. 
See https://github.com/statsmodels/statsmodels/pull/5998

**Comment by bashtage** [comment]
Closes in #5998
